sql/pgsql/_debug_franchiseSeasonRecord_2025_nfl.sql

```
@@ -1,12 +1,16 @@
-select * from public."Franchise" where "Slug" = 'lsu-tigers';
+select * from public."Franchise" where "Slug" = 'cincinnati-bengals';
 
-select * from public."FranchiseSeason" where "FranchiseId" = 'd2ca25ce-337e-1913-b405-69a16329efe7' order by "SeasonYear" desc; -- c13b7c74-6892-3efa-2492-36ebf5220464 2025
+select * from public."FranchiseSeason" where "FranchiseId" = 'be43aed1-bb08-8078-1c9e-8f2a0128061f' order by "SeasonYear" desc; -- 3bf0a8c2-9d7e-06ff-b3ad-a1a0414135d1 2025
 
-select * from public."FranchiseSeasonRecord" where "FranchiseSeasonId" = 'c13b7c74-6892-3efa-2492-36ebf5220464'; -- LSU 2025
+select count(*) from public."FranchiseSeason"
+
+select * from public."FranchiseSeasonRecord" where "FranchiseSeasonId" = '3bf0a8c2-9d7e-06ff-b3ad-a1a0414135d1'; -- Cincy 2025
 
 select * from public."FranchiseSeasonRecord" where "FranchiseId" = 'd2ca25ce-337e-1913-b405-69a16329efe7';
```

SQLFluff (4.3.0)

[error] The 'where' keyword should always start a new line.

(LT14)